<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mxcodecs</class>
 <widget class="QDialog" name="mxcodecs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MX Codecs Installer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageAccept">
      <widget class="QLabel" name="labelAccept">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>481</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;This application allows you to install restricted codecs that permit advanced video and audio functions.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;In some juridictions their distribution may be limited so the user must meet local regulations. &lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Do you assume legal responsibility for downloading these codecs?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageDownload">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Downloading codecs files</string>
         </property>
         <widget class="QLabel" name="labelDownload">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>481</width>
            <height>151</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="buttonBar" columnstretch="0,0,0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="8">
      <widget class="QPushButton" name="buttonCancel">
       <property name="toolTip">
        <string>Cancel any changes then quit</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../usr/share/mx-flash/icons/gtk-cancel.png</normaloff>../../../../../../../../usr/share/mx-flash/icons/gtk-cancel.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+N</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="buttonOk">
       <property name="toolTip">
        <string>Apply any changes then quit</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../usr/share/mx-flash/icons/dialog-ok.png</normaloff>../../../../../../../../usr/share/mx-flash/icons/dialog-ok.png</iconset>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../../../../../usr/share/mx-flash/icons/mxfcelogo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonAbout">
       <property name="toolTip">
        <string>About this application</string>
       </property>
       <property name="text">
        <string>About...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../usr/share/mx-flash/icons/accessories-text-editor.png</normaloff>../../../../../../../../usr/share/mx-flash/icons/accessories-text-editor.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+B</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonHelp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Display help </string>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../usr/share/mx-flash/icons/gnome-help.png</normaloff>../../../../../../../../usr/share/mx-flash/icons/gnome-help.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+H</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>pressed()</signal>
   <receiver>mxcodecs</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
